css/src/components/progress-bar.scss

```diff
@@ -0,0 +1,92 @@
+$progress-bar-height-medium: 0.125rem !default;
+$progress-bar-height-large: 0.25rem !default;
+
+$progress-bar-container-background-color: $secondary-background !default;
+$progress-bar-background-color: $primary !default;
+$progress-bar-success-background-color: $success !default;
+$progress-bar-warning-background-color: $warning !default;
+$progress-bar-danger-background-color: $danger !default;
+$progress-bar-border-radius-rounded: $border-radius-rounded !default;
+
+$progress-bar-animation-duration: 2s !default;
```

Comment:
Nit: especially now that this is put alongside `$progress-bar-transition-duration`, I'm wondering if we should have a more specific name for this. Maybe `$progress-bar-indeterminate-animation-duration`?